### DISCUSSION TITLE
Chinese translation for the document is totally bullsh*t

### DISCUSSION THREAD

**Discussion by bobby-b-song** [BurntSushi/ripgrep#2231] [General] [answered]
As a Chinese, I can certainly tell that it is translated by a machine. It translates `case sensitive`(区分大小写) to `example difference`(案例差异), even the `Donation` part is machine translated. What's more, it seems that it's not accepting PRs, so please just remove it. As a Chinese, I would rather read the English documentation than the Chinese one.

Just to name a few of other translation problems:
`A large number of benchmarks` -> `大量的基准`, which should be `大量的基准测试`
`It is built on top of [Rust's regex engine](https://github.com/rust-lang/regex).` -> `它建在[Rust's regex 引擎](https://github.com/rust-lang-nursery/regex)`, which should be `这个项目基于[Rust 的正则表达式引擎](https://github.com/rust-lang-nursery/regex)`.

**Comment by BurntSushi** [comment, marked as answer]
The README clearly states that the translation is unofficial and may not be up to date. If you can read the English docs, then do so. The presence of the translation doesn't stop you from doing it.

The question here is whether the translation is useful for folks who can't read English. I suspect it is.

**Reply by bobby-b-song** [reply]
Nah, I don't think so. For the `case sensitive` case, first,  I would need to know `案例` has a english translation called `case` and `case` have another meaning that is `capital letter and non-capital letter`. Second, I need to read the Chinese text around it using the `case` word I have "discovered" to infer to the actual meaning of the sentence. For a normal Chinese developer that doesn't have such english skill, it would be a disaster for them to understand.

**Reply by BurntSushi** [reply]
I'm not saying there aren't problems. I'm saying that it's probably better than not having it at all.

**Reply by BurntSushi** [reply]
Basically, if I accept your argument here, then I don't think I could ever link any translation ever, because I wouldn't be able to check whether it contains errors.

**Reply by BurntSushi** [reply]
I think the answer here is for someone to provide a better and more updated translation. Then I can link to that.

**Reply by bobby-b-song** [reply]
Okay, I understand that
